source/util/small_vector.h

```diff
@@ -328,6 +328,14 @@ class SmallVector {
     ++size_;
   }
 
+  void pop_back() {
+    if (large_data_) {
+      large_data_->pop_back();
+    } else {
+      --size_;
```

Comment:
I think you need to call the destructor, as it done in line 93.

Reply:
Extremely embarrassing.  Will add test.